packages/ckeditor5-dev-tests/lib/utils/automated-tests/getkarmaconfig.js

```diff
@@ -39,7 +39,7 @@ module.exports = function getKarmaConfig( options ) {
 		basePath,
 
 		// Frameworks to use. Available frameworks: https://npmjs.org/browse/keyword/karma-adapter
-		frameworks: [ 'mocha', 'sinon-chai' ],
+		frameworks: [ 'mocha', 'sinon-chai', 'webpack' ],
```

Comment:
Why is this change needed? Which library upgrade requires it?

Reply:
See: https://github.com/ryanclark/karma-webpack/commit/8e69ed3c4ce3ec12ff2d86612f05566a704a9b52.